delete_non_clonable.ts

```diff
@@ -88,11 +89,15 @@ export default function deleteNonClonable(obj: any, depth: number = 0) {
     case 'XMLHttpRequest':
       const newXHR: XHRObject = {
         readyState: obj.readyState,
-        responseText: obj.responseText,
         status: obj.status,
         statusText: obj.statusText
       };
 
+      if (obj.responseType === 'arraybuffer' || obj.responseType === 'blob')
```

Comment:
Is there a list of all possible `responseType` somewhere? Otherwise, should we reverse the logic and check for `''` or `text` specifically instead?

Reply:
According to https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/responseType the values can be: `arraybuffer`, `blob`, `document`, `json`, `text`.

We can only access `response` when `arraybuffer` or `blob` otherwise it will trigger an error.